{{ cookiecutter.project_slug }}/backend/requirements.txt

```diff
@@ -1,2 +1,2 @@
--r requirements/unreleased.txt
-src/{{ cookiecutter.python_package_name }}
+-c constraints.txt
+-e src/{{cookiecutter.python_package_name}}
```

Comment:
```suggestion
-c constraints.txt
-e src/{{cookiecutter.python_package_name}}

zope.testrunner
```

Reply:
OK, got it, we need a distinction between dev and deploy.  I'll think about it and come back.